src/ts/components/listselector.ts

```diff
@@ -99,12 +99,13 @@ export abstract class ListSelector<Config extends ListSelectorConfig> extends Co
   }
 
   /**
-   * Adds an item to this selector by appending it to the end of the list of items. If an item with the specified
-   * key already exists, it is replaced.
+   * Adds an item to this selector by doing a sorted insert or by appending the element to the end of the list of items.
+   * If an item with the specified key already exists, it is replaced.
    * @param key the key of the item to add
    * @param label the (human-readable) label of the item to add
+   * @param sortedInsert whether the item should be added respecting the order of keys
    */
-  addItem(key: string, label: LocalizableText) {
+  addItem(key: string, label: LocalizableText, sortedInsert = false) {
```

Comment:
This may be a bit out of scope, if so just stop me :D 

Basically, to have a sorted list, we need to insert _every single item_ with `sortedInsert=true`, otherwise the list won't stay sorted. Considering that, wouldn't it be better to overwrite the `addItem` function in the `AudioTrackSelectBox` subclass? (or `AudioTrackListBox` - not sure how they relate)

Reply:
Let's try!